crates/miner/src/test_helpers.rs

```
@@ -1,3 +1,7 @@
+// TODO: Refactor and remove use of deprecated methods and unused helper
+// functions
+#![allow(deprecated)]
```

Please do not silently ignore deprecated and unused code warnings as they can be easily forgotten. It's best to remove/refactor the underlying issue altogether

---

Hey @nopestack thanks!

I'm still working through this draft while getting familiar with the codebase and I've only silenced them temporarily since my terminal runs out of lines and a lot of the lints are in relation to deprecated code. I went ahead and used the methods that will replace deprecated code if they are available and I will remove the allow attributes once all other lints are addressed.